### ISSUE TITLE
Improved wire handing when using LongConversion and IntConversion

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Wire#135]
Wire to improve the following 

```
      ValueIn symbol = wire.read("symbol");
        sb.setLength(0);
        symbol.text(sb);
```

when using net.openhft.chronicle.wire.LongConversion

**Comment by JerryShea** [comment]
See also #157

**Comment by peter-lawrey** [comment]
This should be fixed, can you clarify if it isn't.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.40](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.40), [BOM-2.20.60](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.60)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
